Tests/Fixtures/app/AppKernel.php

```diff
@@ -70,6 +75,10 @@ public function registerContainerConfiguration(LoaderInterface $loader)
         if (self::VERSION_ID > 30200) {
             $loader->load(__DIR__.'/config/disable_annotations.yml');
         }
+
+        if (self::VERSION_ID < 40200 && file_exists(__DIR__.'/Resources/translations') === false) {
+            self::recurseCopy(__DIR__.'/../translations', __DIR__.'/Resources/translations');
```

Comment:
@stof want ask your help. Is this the correct way to do it? Doing a recursive copy? Thanks for the help!

Reply:
@monteiro, that was written as in SF4.2 was changed default directory structure. If SF will not found translations in new expected place it will throw deprecation message in SF4.2 and fail in SF5.